### DISCUSSION TITLE
Add ability to change username?

### DISCUSSION THREAD

**Discussion by ahstro** [RetroAchievements/RAWeb#792] [Ideas]
Would be a nice feature to have 😊

**Comment by Obspogon** [comment]
agreed

**Comment by WoodenKnees** [comment]
What they waiting for? We are now in 2024 and it is still not implemented.

**Reply by drisc** [reply]
This is being worked on currently, it requires a lot of database changes to properly update things so that we don't end up with orphaned records, we are still uncovering edge cases from when it has been done manually in the past.

Also: Please don't open issues for discussion topics, we are aware of the request, it's in the roadmap, we just haven't driven there yet.

**Comment by Jamiras** [comment]
Implemented in #3071 (and several follow-up PRs).

The actual user name cannot be changed, but the user can specify an alternate display name which is shown everywhere instead of the user name and can be changed periodically.
